lib/muffin_man/catalog_items/base_api.rb

```diff
@@ -20,24 +20,31 @@ class BaseApi < SpApiClient
 
       API_VERSION = "2020-12-01".freeze
 
+      # rubocop:disable Metrics/CyclomaticComplexity
       def search_catalog_items(keywords, marketplace_ids, params = {}, api_version=API_VERSION)
         if sandbox
           keywords = SANDBOX_KEYWORDS
           marketplace_ids = SANDBOX_MARKETPLACE_IDS
           params = {}
         end
         @keywords = keywords.is_a?(Array) ? keywords : [keywords]
+        @identifiers = params["identifiers"].is_a?(Array) ? params["identifiers"] : [params["identifiers"]]
         @marketplace_ids = marketplace_ids.is_a?(Array) ? marketplace_ids : [marketplace_ids]
         @params = params
         @local_var_path = "/catalog/#{api_version}/items"
         @query_params = {
           "marketplaceIds" => @marketplace_ids.join(",")
         }
         @query_params["keywords"] = @keywords.join(",") if @keywords.any?
+        if @identifiers.any?
+          @query_params["identifiers"] = @identifiers.join(",")
+          @query_params["identifiersType"] = params["identifiersType"] || "ASIN"
```

Comment:
Can we put "ASIN" in a constant, maybe something like `DEFAULT_IDENTIFERS_TYPE`?

Reply:
got this done